### PR TITLE
Relax scenario description

### PR DESCRIPTION
**Pull request by FG-TUM**
# Description

- [x] fix padding of domain
- [x] reduce spin dec yaml files to contain only the scenario and no tuning specification
- [x] change md-flex default scenario to super stable periodic particle grid

## ~Resolved Issues~

# How Has This Been Tested?

- [x] Ran some examples
- [x] new tests in MDFlexConfigTest

### CODE REVIEW

**Comment by SteffenSeckler** [comment]
tests fail

File: examples/md-flexible/input/SpinodalDecomposition.yaml [hunk 1]
```diff
@@ -1,20 +1,8 @@
...
-tuning-max-evidence              :  10
 functor                          :  Lennard-Jones (12-6)
-newton3                          :  [disabled, enabled]
 cutoff                           :  2.5
-box-min                          :  [-0.25, -0.25, -0.25]
-box-max                          :  [241.25, 241.25, 241.25]
-cell-size                        :  [0.34-2]
+# these are the box size values for a equilibration with 160x160x160 and spacing 1.5
+box-min                          :  [-0.75, -0.75, -0.75]
+box-max                          :  [239.25, 239.25, 239.25]
```
Comment (SteffenSeckler) [inline examples/md-flexible/input/SpinodalDecomposition.yaml:5]: why did you change these?

File: examples/md-flexible/input/SpinodalDecomposition_equilibration.yaml [hunk 2]
```diff
@@ -1,20 +1,5 @@
...
-tuning-strategy                  :  active-harmony
-tuning-interval                  :  1000
-tuning-samples                   :  3
-tuning-max-evidence              :  10
 functor                          :  Lennard-Jones (12-6)
-newton3                          :  [enabled, disabled]
 cutoff                           :  2.5
-box-min                          :  [-0.25, -0.25, -0.25]
-box-max                          :  [241.25, 241.25, 241.25]
-cell-size                        :  [0.34-2]
```
Comment (SteffenSeckler) [inline examples/md-flexible/input/SpinodalDecomposition_equilibration.yaml:17]: why are these not needed anymore?

Comment (FG-TUM) [reply examples/md-flexible/input/SpinodalDecomposition_equilibration.yaml:17]: The box size is calculated from the grid object specified further below.
All other options do not directly specify the scenario but the algorithm configuration or tuning behavior. Since these yaml files should describe only the scenario I deleted those options.

File: examples/md-flexible/input/SpinodalDecomposition.yaml (hunk 1 above)
Comment (FG-TUM) [reply examples/md-flexible/input/SpinodalDecomposition.yaml:5]: This is the new (and correct) size when initializing the equilibration with a grid of 160x160x160.
